src/framework/filterControl/forms/MultipleSelectFilterForm.spec.jsx

```diff
@@ -4,10 +4,15 @@ import MultipleSelectFilterForm from './MultipleSelectFilterForm.jsx';
 import {
   Filter,
   FilterOption,
+  OneOfOption,
 } from '../../services';
 
-describe('MultipleSelectFilterForm', () => {
-  const options = ['Active', 'Archived', 'Stopped'];
+fdescribe('MultipleSelectFilterForm', () => {
```

Comment:
`fdescribe`. Then `'returns joined string with ONE_OF comparisonType'` test fails in `Filter.spec.js` (fixed by initializing `comparisonValue` to `[new OneOfOption(5), new OneOfOption(6)]`).